BEE-49248: Process reservations under project (#113)
electric-cloud/EC-DslDeploy: help/help.adoc

[[releaseNotes]]
== DslDeploy plugin release notes

=== 4.3.2

> * Process reservations under project. (Note that previously exported DSLs will need to be re-exported for import to succeed)

Reviewer: ```suggestion
* Process reservations under project. 
+
NOTE: Previously exported DSLs must be re-exported for import to succeed.
```
